CHANGELOG.md

```diff
@@ -2,6 +2,55 @@
 
 [Releases](https://github.com/NetApp/harvest/releases)
```

Comment:
missing 
1692df1 feat: supporting new_status metric in native dashboards
29fb793 refactor: use table for harvest status

Reply:
first one is captured in 
> - New plugin [value_to_map](https://github.com/NetApp/harvest/blob/main/cmd/poller/plugin/README.md#value_to_num) helps map labels to numeric values for Grafana dashboards. Current dashboards updated to use this plugin [#319](https://github.com/NetApp/harvest/pull/319)

second not worth mentioning